src/main/kotlin/com/terraformation/backend/report/db/ReportStore.kt

```diff
@@ -380,6 +387,62 @@ class ReportStore(
     }
   }
 
+  /** Save the seed bank buildStartDate, buildCompletedDate, and operationStartDate */
+  private fun saveSeedBankInfo(body: ReportBodyModel) {
+    val reportBody = body.toLatestVersion()
+    reportBody.seedBanks
+        .filter { it.selected }
+        .forEach {
+          val seedBank = facilitiesDao.fetchOneById(it.id)
```

Comment:
Question: does #L213 imply this is a transaction as well?
If we don't find the seedBank here do we want to throw an exception `?: throw FacilityNotFoundException` or let it pass (we won't end up updating). The current implementation does seem better than throwing an exception (otherwise end-user will have to go through hoops to clean up the report). Also, a very unlikely scenario.

Reply:
Yes, it is in a transaction—I hadn't realized that at first. I think not throwing an exception is probably best like you say.